### PR TITLE
augustus: Fix missing library: libbamtools.so.2.4.0 [Linux]

### PR DESCRIPTION
**Pull request by sjackman**
```
Missing libraries:
  libbamtools.so.2.4.0
libbamtools.so.2.4.0
  libexec/bin/bam2hints
  libexec/bin/filterBam
```

### CODE REVIEW

File: Formula/augustus.rb [hunk 1]
```diff
@@ -3,13 +3,13 @@ class Augustus < Formula
   homepage "http://bioinf.uni-greifswald.de/augustus/"
   url "http://bioinf.uni-greifswald.de/augustus/binaries/augustus-3.3.tar.gz"
   sha256 "b5eb811a4c33a2cc3bbd16355e19d530eeac6d1ac923e59f48d7a79f396234ee"
+  revision 1 if OS.linux?
```
Comment (iMichka) [inline Formula/augustus.rb]: I always use `unless OS.mac?`

File: Formula/augustus.rb [hunk 2]
```diff
@@ -19,6 +19,7 @@ def install
     inreplace "Makefile", %r{ln -sf.*/usr/local/bin/}, "#ln -sf"
 
     # Compile executables for macOS. Tarball ships with executables for Linux.
+    system "make", "clean"
```
Comment (iMichka) [inline Formula/augustus.rb]: `unless OS.mac?` to help with the merges. Not sure why you need the call to clean here?

File: Formula/augustus.rb (hunk 1 above)
Comment (sjackman) [reply Formula/augustus.rb]: Cool. I'll change it.

File: Formula/augustus.rb (hunk 2 above)
Comment (sjackman) [reply Formula/augustus.rb]: Will do.
The source tarball includes precompiled binaries for Linux, and two of these executables depend on the library `libbamtools.so.2.4.0`. The brewed version of `bamtools` provides not this version but `libbamtools.so.2.5.1`, which causes `brew linkage` to fail.

File: Formula/augustus.rb [hunk 3]
```diff
@@ -3,22 +3,36 @@ class Augustus < Formula
...
   def install
+    unless OS.mac?
+      # Fix error: api/BamReader.h: No such file or directory
+      inreplace "auxprogs/bam2hints/Makefile",
+        "INCLUDES = /usr/include/bamtools",
+        "INCLUDES = #{Formula["bamtools"].include/"bamtools"}"
+      inreplace "auxprogs/filterBam/src/Makefile",
+        "BAMTOOLS = /usr/include/bamtools",
+        "BAMTOOLS= #{Formula["bamtools"].include/"bamtools"}"
+
```
Comment (iMichka) [inline Formula/augustus.rb]: end?

Comment (sjackman) [reply Formula/augustus.rb]: Woops 🙈

Comment (sjackman) [reply Formula/augustus.rb]: Fixed

**Comment by sjackman** [comment]
Thanks for merging, Michka!

**Pull request closed without merging**